code/game/machinery/telecomms/ntsl2.dm

```diff
@@ -183,7 +248,7 @@ GLOBAL_DATUM_INIT(nttc_config, /datum/nttc_configuration, new())
 		var/new_message = original
 		for(var/reg in regex)
 			var/replacePattern = pencode_to_html(regex[reg])
-			var/regex/start = regex(reg, "gi")
+			var/regex/start = regex("\\b[reg]\\b", "gi")
```

Comment:
Do not hard code this.

Pulling things like AI out of Maint is the problem of the script, not the hardcoded regex, unless it doesn't work in script.

Reply:
Are you **really** going to assume everyone who can click buttons in a UI is going to know how to operate regex properly? This is straight up required to stop this happening

[!Image]
(https://user-images.githubusercontent.com/25063394/48642445-c7400400-e9d4-11e8-86fe-be51df87dc2a.png)

Currently you can have a simple thing to replace RD with research director or CE with chief engineer. But that will break the font line since its using *<font fa**ce**=Ve**rd**ana>*. I would like this to stay for the stated reasons.

Reply:
After testing, the `\\b` is staying on the regex. Even after removing the `<font>` that pencode adds, the issue still persists.

With regex correction:
![image](https://user-images.githubusercontent.com/25063394/48670992-75ed5d00-eb19-11e8-87fc-f22a7398b8a1.png)

Without regex correction:
![image](https://user-images.githubusercontent.com/25063394/48670993-7e459800-eb19-11e8-9a99-b3a735246cc5.png)